networks/p2p/peer_test.go

```diff
@@ -87,7 +99,7 @@ func testPeerWithRWs(protos []Protocol, channelSize int) (func(), []*conn, *Peer
 	}
 
 	peer, _ := newPeer(serverSideConn, protos, defaultRWTimerConfig)
-	errc := make(chan error, 1)
+	errc := make(chan error, 2)
```

Comment:
I cannot see why the size of this channel should be 2.. could you explain please?

Reply:
It was my mistake. There's no reason to change the size of the err channel. Applied at here https://github.com/klaytn/klaytn/pull/1954/commits/407e6c73512663a98a0edc7fe9a7bbac9d64797e.